### PR TITLE
LP-2631 [Fix Bug] Add model for webinar event

### PR DESCRIPTION
**Pull request by javeriaejaz-arbisoft**

### CODE REVIEW

File: openedx/adg/lms/webinars/tests/test_models.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+from openedx.adg.lms.webinars.tests.factories import WebinarFactory
+
+
+@pytest.mark.django_db
+@pytest.fixture(name='webinar')
+def webinar_fixture():
+    """
+    Create webinar fixture to use as a parameter to other pytests or fixtures
+    """
+    return WebinarFactory()
```
Comment (asadali145) [inline openedx/adg/lms/webinars/tests/test_models.py:19]: I guess we don't need this fixture or do we? We can directly use `WebinarFactory()` in tests.

File: openedx/adg/lms/webinars/tests/test_models.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+
+@pytest.mark.django_db
+def test_invalid_start_date_and_end_date_in_webinar(webinar):
```
Comment (asadali145) [inline openedx/adg/lms/webinars/tests/test_models.py:23]: Can we also add test for valid case? Same for the banner.

File: openedx/adg/lms/webinars/models.py [hunk 3]
```diff
@@ -75,7 +75,8 @@ def __str__(self):
         return self.title
 
     def clean(self):
-        if self.start_time > self.end_time:
+        super().clean()
+        if self.start_time and self.end_time and self.start_time > self.end_time:
```
Comment (asadali145) [inline openedx/adg/lms/webinars/models.py:79]: Can we check that the dates are in future.

File: openedx/adg/lms/webinars/models.py [hunk 4]
```diff
@@ -75,7 +75,8 @@ def __str__(self):
...
             raise ValidationError({'start_time': _('End date must be greater than start date')})
         if self.banner:
             error = validate_file_size(self.banner, BANNER_MAX_SIZE)
```
Comment (MunamMubashir-arbisoft) [inline openedx/adg/lms/webinars/models.py:82]: We can move this to `clean_banner`

File: openedx/adg/lms/webinars/models.py (hunk 3 above)
Comment (MunamMubashir-arbisoft) [inline openedx/adg/lms/webinars/models.py:79]: We can add future date checks for `start_time` and `end_time`  in `clean_start_date` and `clean_end_date`

**Review by MunamMubashir-arbisoft — CHANGES_REQUESTED** [review]

File: openedx/adg/lms/webinars/tests/test_models.py (hunk 1 above)
Comment (javeriaejaz-arbisoft) [reply openedx/adg/lms/webinars/tests/test_models.py:19]: It is more neat than initializing the factory in every test case.

**Comment by zahid-arbisoft** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/1441/workflows/workflows_id/jobs/1441/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/webinars/models.py (100%)
openedx/adg/lms/webinars/tests/factories.py (100%)
openedx/adg/lms/webinars/tests/test_models.py (100%)
-------------
Total:   59 lines
Missing: 0 lines
Coverage: 100%
-------------
```

**Review by harris-irfan — APPROVED** [review]
LGTM 👍

**Review by asadali145 — APPROVED** [review]

**Review by MunamMubashir-arbisoft — APPROVED** [review]
Great work!

**Pull request merged**